tests/test_port_dpb_neigh.py

```diff
@@ -0,0 +1,313 @@
+from swsscommon import swsscommon
+import redis
+import time
+import os
+import pytest
+import json
+import re
+from port_dpb import DPB
+
+@pytest.mark.usefixtures('dpb_setup_fixture')
+class TestPortDPBNeigh(object):
+    def clear_srv_config(self, dvs):
+        dvs.servers[0].runcmd("ip address flush dev eth0")
+        dvs.servers[1].runcmd("ip address flush dev eth0")
+
+    """
+    Test port neighbor dependency for Dynamic port breakout
+    - Add neighbors on port
+    - Port should be able to be deleted(breakout) without dependency on neighbor on the port
+    """
+    def test_set_neighbor_DPB(self, dvs, testlog):
+        dvs.setup_db()
+        self.clear_srv_config(dvs)
+
+        # bring up interface
+        dvs.set_interface_status("Ethernet0", "up")
+
+        # create interface
+        dvs.create_l3_intf("Ethernet0", "")
+
+        # assign IP to interface
+        dvs.add_ip_address("Ethernet0", "10.0.0.1/24")
+
+        # add neighbor
+        dvs.set_neighbor("Ethernet0", "10.0.0.2", "00:01:02:03:04:05")
+
+        # check application database
+        # check that the FDB entries were inserted into State DB for Ethernet64, Ethernet68 with Vlan6
+        ok, extra = dvs.is_table_entry_exists(dvs.pdb, "NEIGH_TABLE",
```

Comment:
why do we have to pass dvs.pdb?

Reply:
This test case,  neighbor is added to config db,  Check Neighbor is in NEIGH_TABLE in app db,  later check the neighbor is deleted after interface is deleted and port can be deleted with no dependency.